blog/journey-to-decentralization.md

```diff
@@ -0,0 +1,100 @@
+---
+slug: journey-to-decentralization
+hide_table_of_contents: false
+title: "XMTP: The journey to decentralization"
```

Comment:
Changed this from "Our journey to decentralization" to "XMTP: The journey to decentralization" - just to shift the focus to the journey being that of XMTP - and not the journey of XMTP Labs.